README.rst

```diff
@@ -107,10 +107,20 @@ data-taking years (e.g. 2011).
 
 Example
 ---------
-The following will generate the workflow to run the example for a given record
+Before running example, you might want to install necessary packages:
+
+.. code-block:: console
+
+    $ # create new virtual environment
+    $ virtualenv ~/.virtualenvs/myreana
+    $ source ~/.virtualenvs/myreana/bin/activate
+    $ # install reana-commons and reana-client
+    $ pip install git+git://github.com/reanahub/reana-demo-cms-reco.git@master#egg=cms-reco
```

Comment:
Please edit `./run-tests.sh`, there are sphinx bits left there.

Reply:
I modified it. `./run-tests.sh` even when using `python setup.py`, still now running the tests.

Reply:
Yes, I see. We can merge this and revive tests later, since this commit modifies only sphinx and does not alter anything regarding non-running tests...